bindings/java/src/main/com/apple/foundationdb/Tenant.java

```diff
@@ -290,9 +290,11 @@ default CompletableFuture<byte[]> purgeBlobGranules(byte[] beginKey, byte[] endK
 
 
 	/**
-	 * Runs {@link #waitPurgeGranulesComplete(Function)} on the default executor.
+	 * Runs {@link #waitPurgeGranulesComplete(byte[] purgeKey)} on the default executor.
 	 *
 	 * @param purgeKey key to watch
+
+	 * @return -- undocumented
```

Comment:
It returns void here. If it returns successfully, the data has been purged tracked by the provided purgeKey from purgeBlobGranules().